src/inputFile.h

```diff
@@ -188,6 +188,7 @@ class InputFile
 
     T_F_FLOAT neighbor_skin;
     int neighbor_type;
+    T_INT max_neighbor;
```

Comment:
The variable name is misleading.  It is not the maximum number of neighbors, it is a guess on that number that is used to pre-allocate memory.

Reply:
Yeah, `max_neigh_guess` is better